### PR TITLE
Use #[doc(cfg)]

### LINKED ISSUE #835: Use #[doc(cfg)] instead of manually indicating `feature = "blah"` in the comments
Apparently there's a way to do this. Not exactly sure on the details, though... It's unstable, so maybe we'd need to work around that somehow too...

### PR DESCRIPTION
**Pull request by gwenn**
Fix #835

### CODE REVIEW

File: src/load_extension_guard.rs [hunk 1]
```diff
@@ -14,6 +14,7 @@ use crate::{Connection, Result};
 ///     conn.load_extension(Path::new("my_sqlite_extension"), None)
 /// }
 /// ```
+#[cfg_attr(docsrs, doc(cfg(feature = "load_extension")))]
```
Comment (gwenn) [inline src/load_extension_guard.rs:17]: Works only when put here (not in lib.rs)...

File: src/hooks.rs [hunk 2]
```diff
@@ -9,11 +9,12 @@ use crate::ffi;
 
 use crate::{Connection, InnerConnection};
 
-/// `feature = "hooks"` Action Codes
+/// Action Codes
 #[derive(Clone, Copy, Debug, PartialEq)]
 #[repr(i32)]
 #[non_exhaustive]
 #[allow(clippy::upper_case_acronyms)]
+#[cfg_attr(docsrs, doc(cfg(feature = "hooks")))]
```
Comment (gwenn) [inline src/hooks.rs:17]: Works only when put here (not in lib.rs)...

File: src/types/to_sql.rs [hunk 3]
```diff
@@ -70,9 +72,11 @@ from_value!(Vec<u8>);
 // `i128` needs in `Into<Value>`, but it's probably fine for the moment, and not
 // worth adding another case to Value.
 #[cfg(feature = "i128_blob")]
+#[cfg_attr(docsrs, doc(cfg(feature = "i128_blob")))]
```
Comment (gwenn) [inline src/types/to_sql.rs:75]: Doesn't work (due to macro ?)

File: src/types/to_sql.rs [hunk 4]
```diff
@@ -70,9 +72,11 @@ from_value!(Vec<u8>);
...
 from_value!(i128);
 
 #[cfg(feature = "uuid")]
+#[cfg_attr(docsrs, doc(cfg(feature = "uuid")))]
```
Comment (gwenn) [inline src/types/to_sql.rs:79]: Doesn't work (due to macro ?)

**Review by thomcc — APPROVED** [review]
Sorry for the delay. This seems fine.

**Pull request merged**